### ISSUE TITLE
Need a script for building API docs of Twisted for releases

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3637]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#3637|
|Type|enhancement|
|Created|2009-01-31 17:01:22Z|
|Branch|https://github.com/twisted/twisted/tree/api-script-3637|

Right now there's some Python library functionality for building API docs for Twisted, but it's not exposed to the command line. It should be done so, and the command line tool should encode the policy itself so the user doesn't need to provide boring details like "project name" and "source link URL".

<details><summary>Searchable metadata</summary>

```
trac-id__3637 3637
type__enhancement enhancement
reporter__radix radix
priority__normal normal
milestone__totally_automated_release_infrastructure totally automated release infrastructure
branch__branches_api_script_3637 branches/api-script-3637
branch_author__radix radix
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1233421282000000 1233421282000000
changetime__1233449614000000 1233449614000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_1"></a>|@radix commented|
|-|-|

(In [26138]) Branching to 'api-script-3637'

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_2"></a>|@radix removed owner|
|-|-|

Ready for review.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph set owner to @radix|
|-|-|

Looks good to me!  Go ahead and merge.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_4"></a>|@radix set status to `closed`|
|-|-|

(In [26148]) Merge api-script-3637

Author: radix
Reviewer: glyph
Fixes: [#3637](https://github.com/twisted/twisted/issues/3637)

There is now a command line tool for generating the API docs of Twisted.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
